java/pending_code_changes.txt

```diff
@@ -1 +1,5 @@
-
+Code changes:
+ - Using new possibleLengthInfo to decide whether a short number is the right
+length or not. This could result in more specific results; whereas before, a
```

Comment:
I would do x.y.z because it's like a metadata change: no one can or should change code because of it.